Assets/ShooterProject/Scripts/Player Movement/UpdateCharacterController.cs

```diff
@@ -0,0 +1,12 @@
+namespace UnityEngine.XR.Interaction.Toolkit
+{
+    public class UpdateCharacterController : CharacterControllerDriver
+    {
+        #region LifeCycle
+        private void FixedUpdate()
+        {
+            UpdateCharacterController();
+        }
+        #endregion
```

Comment:
Внутри регионов делай пустые строки
```suggestion
        #region LifeCycle
        
        private void FixedUpdate()
        {
            UpdateCharacterController();
        }
        
        #endregion
```